### ISSUE TITLE
PGCR timePlayedSeconds seem to be in int16

### ISSUE THREAD

**Issue by BinarMorker** [Bungie-net/api#741]
I'm getting a lot of 32767, which could be coincidental. However, here is a quick list of the duplicate timePlayedSeconds values I'm getting:

```
[ { value: '32767', count: 92 },
        { value: '687', count: 3 },
        { value: '905', count: 3 },
        { value: '48', count: 3 },
        { value: '50', count: 3 },
        { value: '887', count: 3 },
        { value: '813', count: 3 },
        { value: '76', count: 3 },
        ... ]
```

There is no way this value at the top is a random occurrence. Is it possible to display the timePlayedSeconds as a int32 at least? Or are those values already saved as is in your databases?

It's a bummer because it's skewing my data for players who spent a lot of time in certain activities, like raids for example.

**Comment by BinarMorker** [comment]
Hi, has there been any findings on this?

**Comment by vthornheart-bng** [comment]
Indeed, it looks like TimePlayedSeconds is being stored as an int16 in player activity data.

This means any existing records will not be salvageable, and I'm also uncertain if/when this will be fixed in the API itself, as it requires a change to the underlying stats system on which we rely (which is on game deployment cadence, and with game deployment time constraints).  I'll keep you posted once I get more info from them.

**Comment by vthornheart-bng** [comment]
I filed a bug in our bug tracking system for them to determine when they can get to it.  I'll let you know once I get more info.
